src/cli.ts

```diff
@@ -21,10 +32,93 @@ prog.command('dev')
 		'dev-port': number,
 		live: boolean,
 		hot: boolean,
-		bundler?: string
+		bundler?: 'rollup' | 'webpack'
 	}) => {
-		const { dev } = await import('./cli/dev');
-		dev(opts);
+		const cwd = path.resolve(process.env.SAPPER_BASE || '');
+
+		const { dev } = await import('./api/dev');
+
+		try {
+			const watcher = dev({
+				cwd,
+				port: opts.port,
+				'dev-port': opts['dev-port'],
+				live: opts.live,
+				hot: opts.hot,
+				bundler: opts.bundler
+			});
+
+			let first = true;
+
+			watcher.on('ready', async (event: ReadyEvent) => {
+				if (first) {
+					console.log(colors.bold.cyan(`> Listening on http://localhost:${event.port}`));
+					if (opts.open) {
+						const { exec } = await import('child_process');
+						exec(`open http://localhost:${event.port}`);
+					}
+					first = false;
+				}
+
+				// TODO clear screen?
```

Comment:
Yeah, but I would [soft-clear](https://github.com/lukeed/console-clear#clearkeephistory) here. I've found that whenever I do a hard clear I get some push back.

Reply:
Oh I didn't know about console-clear — handy, thanks. This is an old TODO though (I just copied it from a now-deleted file), should probably be dealt with in a separate PR